ada/ast.py

```diff
@@ -7242,6 +7252,15 @@ class Pragma(AdaNode):
 
     xref_entry_point = Property(True)
 
+    @langkit_property(public=True)
+    def is_ghost_code():
+        """
+        Return whether this declaration is ghost code or not. See SPARK RM 6.9.
+        """
+        return Entity.id.sym.any_of(
+            'Assert', 'Assume'
```

Comment:
Missing `Loop_Invariant`, maybe also `Invariant` (for types), `Pre`, `Post`? (and their 'Class variant?)